internal/backend/httpsvc/intercept.go

```diff
@@ -66,42 +67,36 @@ func intercept(z *zap.Logger, cfg *LoggerConfig, extractors []RequestLogExtracto
 
 		w.Header().Set("X-AutoKitteh-Process-ID", fixtures.ProcessID())
 
-		z := z.With(zap.String("method", r.Method), zap.String("path", r.URL.Path))
-		msg := fmt.Sprintf("HTTP request: %s %s", r.Method, r.URL.Path)
-
-		level := cfg.ImportantLevel.Level()
-		if unimportant.MatchString(r.URL.Path) {
-			level = cfg.UnimportantLevel.Level()
-		}
-
-		if ce := z.Check(level, msg); ce != nil {
-			var fields []zap.Field
-			for _, x := range extractors {
-				fields = append(fields, x(r)...)
-			}
-
-			ce.Write(fields...)
-		}
-
-		rwi := &responseInterceptor{ResponseWriter: w, StatusCode: http.StatusOK, logger: z}
+		l := z.With(zap.String("method", r.Method), zap.String("path", r.URL.Path))
+		rwi := &responseInterceptor{ResponseWriter: w, StatusCode: http.StatusOK, logger: l}
 
 		w.Header().Set("Trailer", "X-AutoKitteh-Duration")
 
-		t0 := time.Now()
-		r = r.WithContext(context.WithValue(r.Context(), t0CtxKey, t0))
+		startTime := time.Now()
+		r = r.WithContext(context.WithValue(r.Context(), startTimeCtxKey, startTime))
 
 		next.ServeHTTP(rwi, r)
-		d := time.Since(t0)
+		d := time.Since(startTime)
 
 		w.Header().Add("X-AutoKitteh-Duration", d.Truncate(time.Microsecond).String())
 
+		level := cfg.ImportantLevel.Level()
```

Comment:
Not sure what are you trying to do here

Could you elaborate?

Reply:
The diff looks confusing for no reason, the best way to review this PR is to compare the entire function before and after - and then you see that I basically cancelled the first log entry, and moved its fields to the second log entry.

Reply:
I updated the PR description with relevant links